File: DrQuine/Colleen.c
/*
  Commentaire hors fonction
 */

#include <stdio.h>

void funcplus(){}

int main()
{
	char a[]="/*%c%cCommentaire hors fonction*/#include <stdio.h> void funcplus(){};%cint main(){char*a=%c%s%c;printf(a,10,34,a,34);/*Salut Colleen dans le main*/}";printf(a,10,34,a,34);
	/*
	  Salut Colleen dans le main
	 */
}

File: DrQuine/Bonus/Colleen.py
#Commentaire hors fonction
def funcplus():
	return;
def main():
	string = "#Commentaire hors fonction%cdef funcplus():%c%creturn;%cdef main():%c%cstring = %c%s%c%c%cprint(string %c(10,10,9,10,10,9,34,string,34,10,9,37,10,9,10,10,9))%c%c#Commentaire dans fonction%cif __name__ == '__main__':%c%cmain()"
	print(string %(10,10,9,10,10,9,34,string,34,10,9,37,10,9,10,10,9))
	#Commentaire dans fonction
if __name__ == '__main__':
	main()
